src/printer/printer.cpp

```diff
@@ -117,5 +117,19 @@ Printer* Printer::getPrinter(OutputLanguage lang)
   return d_printers[lang].get();
 }
 
+void printUnknown(std::ostream& out, const std::string& name)
```

Comment:
I would call this `printUnknownCommand`.

Reply:
Done!